unused/icshandler/IcsHandler.java

```diff
@@ -16,6 +16,10 @@
 import java.io.IOException;
 import java.util.GregorianCalendar;
 
+//@@author kyhjonathan-unused
+//Unused as there were too many issues with the dependacies of the library
+//and the library itself was not very well documented
+//Decided to half development of ics and focuses on testing and documentation of other features
 public class IcsHandler {
```

Comment:
yes, you should definitely be credited for attempting to incorporate the ICS handler!